releases_sort.go

```diff
@@ -1,7 +1,27 @@
 package versionbundle
 
+import "time"
+
 type SortReleasesByVersion []Release
 
 func (r SortReleasesByVersion) Len() int           { return len(r) }
 func (r SortReleasesByVersion) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
 func (r SortReleasesByVersion) Less(i, j int) bool { return r[i].Version() < r[j].Version() }
+
+type SortReleasesByTimestamp []Release
+
+func (r SortReleasesByTimestamp) Len() int      { return len(r) }
+func (r SortReleasesByTimestamp) Swap(i, j int) { r[i], r[j] = r[j], r[i] }
+func (r SortReleasesByTimestamp) Less(i, j int) bool {
```

Comment:
Timestamps must be parsed to proper `time.Time` entries due to possibility of different time zones in different releases.